app/assets/javascripts/discourse/app/lib/hashtag-autocomplete.js

```diff
@@ -13,27 +13,23 @@ import {
 import { search as searchCategoryTag } from "discourse/lib/category-tag-search";
 
 export function setupHashtagAutocomplete(
-  context,
+  orderedContextTypes,
```

Comment:
maybe `hashtagConfiguration` ? I have not much idea what is `orderedContextTypes` from reading it

Reply:
Yes that is fair enough, it evolved a little over time, `contextualHashtagConfiguration` makes more sense :+1: